### PR TITLE
Add offer history item

### PR DESCRIPTION
**Pull request by ds300**
https://artsyproduct.atlassian.net/browse/PURCHASE-667
https://zpl.io/2yy9J5y

seller s offer

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 50bd090486085ed445e2134bcab7c2fc64c163f5

https://deploy-preview-1610--artsy-reaction.netlify.com

**Comment by ds300** [comment]
I had to fix a couple of things after submitting this PR, but this is ready for review again :pray:

File: src/Apps/Order/Components/OfferHistoryItem.tsx [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+  { showingFullHistory: boolean }
+> {
+  state = { showingFullHistory: false }
+  render() {
+    const {
+      order: { totalListPrice, lastOffer, offers: _offers, buyer },
+      ...others
+    } = this.props
+    const { showingFullHistory } = this.state
+    const offers = _offers.edges.filter(
```
Comment (sweir27) [inline src/Apps/Order/Components/OfferHistoryItem.tsx]: Ah-- neat trick casting this as `_offers`. Could we either add a comment explaining what this filter is doing or rename the variable to `pastOffers` or something so it's clear?

**Comment by ds300** [comment]
> It appears to be undoing the changes in generated relay files from the last PR I did. Any idea why that is?

No idea. I merged master in and ran `yarn relay` and this is what I got. Will investigate.

**Comment by ashkan18** [comment]
@ds300 https://github.com/artsy/metaphysics/pull/1404 is merged if you get latest MP schema you can use `fromParticipant` field on `Offer`, we can do that in a separate PR too.

**Pull request merged**